WIP: Simplify IG charter as suggested by Kaz

mkovatsc opened the pull request:
Dear group

I made another iteration over the charter as discussed during the call on [22 May 2019](https://www.w3.org/2019/05/22-wot-minutes.html):

* I simplified the scope
* I removed concrete deliverables
* I updated the coordination section (mainly removed closed groups)
* I included wording proposals from https://github.com/w3c/wot/issues/599#issuecomment-447347982

I still need feedback about:

* current list of co-chair candidates
* currently still active IG documents (e.g., @mmccool living security document?)
* checking the coordination section and add new groups (@ashimura , can you help?)
* @mlagally please check the scope, as you had the "hardest feelings" about some topics
* should we list Thing Directory now and remove it, if it ends up in the WG charter, or add it later if it does not?

k-toumura commented:
[Rendered version](https://cdn.statically.io/gh/w3c/wot/4ce75d2/charters/wot-ig-2019.html). 
(using commit hash as URL to avoid cache.  When you make a further commit, please regenerate URL)

mkovatsc commented:
Thanks, @k-toumura !

ashimura commented:
looks good to me :) Thanks a lot @mkovatsc !

BTW, it seems there is some extra text saying:
[[
 As of the day of chartering, the WoT Interest Group is working on the following documents:

    Deliverable 1 — description
]]
but maybe we can simply remove it, can't we?

mmccool commented:
Sorry for my late response on this.   As I noted in email, the wot-security documents and the related docs wot-security-best-practices and wot-security-testing-plan need to be constantly updated to maintain relevance.  As these are Notes maintaining these as “living documents” can be performed by the IG.

Other than that, I would add “OCF” and “OGS” to the list of organizations we have collaborated/interacted with.   These are not critical, but OGS in particular we should include if there is still time for edits.

Also, I am happy to help chair the IG, but please use camel case for my last name: McCool

For clarity, it would be willing to accept the document as it stands and forward it to the W3M, but if there is still time I would add “Maintaining living documents on security” as one of the deliverables.

mkovatsc commented:
[New preview snapshot](https://cdn.staticaly.com/gh/w3c/wot/7cdc8517f38cf3a979809384f44370f27f692a8a/charters/wot-ig-2019.html)

* Added OGC
* Added "test cases" to possible other documents to include security test cases
* Removed list for active documents
* Fixed McCool casing

mmccool commented on charters/wot-ig-2019.html:
```diff
@@ -167,278 +179,202 @@ <h1 id="title">Proposed Web of Things Interest Group Charter</h1>
+          and is intended to complement the role of the Web of Things Working Group (WoT WG).
+          In particular, the WoT IG will propose new work items based upon new requirements, use cases,
+          and discussions with IoT developer communities, industry alliances, and standards development organizations.
+          Vice versa, the WoT IG will consider new topics that are submitted to it by the WoT WG.
```
Vice versa -> Conversely,

mmccool commented on charters/wot-ig-2019.html:
```diff
@@ -167,278 +179,202 @@ <h1 id="title">Proposed Web of Things Interest Group Charter</h1>
+          <ul>
+            <li>use case and requirement documents;</li>
+            <li>experience and/or implementation reports for identified building blocks;</li>
+            <li>primer, best practice, or test case documents to support WoT developers when designing applications and/or Things.</li>
```
Maybe add "including for security."   Although, leaving it out is probably also ok.

mmccool commented on charters/wot-ig-2019.html:
```diff
@@ -167,278 +179,202 @@ <h1 id="title">Proposed Web of Things Interest Group Charter</h1>
+            <dt><a href="https://echonet.jp/english/">ECHONET Consortium</a></dt>
+            <dd>For collaboration on integrating ECHONET Consortium based platforms within the Web of Things, including platform metadata and approaches for enabling semantic interoperability, and end to end security across platforms.</dd>
+            <dt><a href="http://www.iiconsortium.org">Industrial Internet Consortium</a></dt>
+            <dd>For collaboration on a testbed for the Web of Things.</dd>
```
Expand IIC description, it's not just testbeds:
For collaboration on developing application testbeds and gathering user requirements for the Web of Things

mmccool commented on charters/wot-ig-2019.html:
```diff
@@ -167,278 +179,202 @@ <h1 id="title">Proposed Web of Things Interest Group Charter</h1>
+            <dt><a href="https://www.omaspecworks.org">OMA SpecWorks</a></dt>
+            <dd>For collaboration on integrating LWM2M based platforms with the Web of Things.</dd>
+            <dt><a href="http://www.opengeospatial.org/">Open Geospatial Consortium</a></dt>
+            <dd>For coordination on the OGC SensorThings API.</dd>
```
For coordination on the OGC SensorThings API and location metadata

mmccool commented on charters/wot-ig-2019.html:
```diff
@@ -167,278 +179,202 @@ <h1 id="title">Proposed Web of Things Interest Group Charter</h1>
+            <dt><a href="http://www.gsma.com">GSMA</a></dt>
+            <dd>For coordination in respect to the interests of Network Operators.</dd>
+          </dl>
+        </section>
```
Part of me wants to add a few more, like EdgeX and OpenAPI, but as long as this is meant to be an open-ended list, this is good enough.

mmccool commented on charters/wot-ig-2019.html:
```diff
@@ -167,278 +179,202 @@ <h1 id="title">Proposed Web of Things Interest Group Charter</h1>
-            <dd><i class="todo">[specific nature of liaison]</i></dd>
-          </dl>
 
+          <p>To succeed in establishing inter-platform standards, W3C needs to coordinate with IoT alliances and standards development organizations. The following provides examples of organizations with which the Interest Group has already had significant coordination, during the period of the first charter. A <a href="https://www.w3.org/WoT/IG/wiki/How_does_W3C_compare_to_other_organizations">l ...
```
The following list provides
(using "the following" as a noun is technically correct but sounds odd)
coordination, during -> coordination during (eliminate comma splice)

mkovatsc replied:
Interestingly, this was from the boilerplate text..

mmccool commented:
I've reviewed and accepted this version, but I did suggest a few small changes.   The role of IIC should be expanded slightly.  I see OCF and OGC are mentioned already.

mlagally commented on charters/wot-ig-2019.html:
```diff
@@ -457,37 +393,29 @@ <h2 id="participation">
+        <h2>Communication</h2>
 
-    <p id="public">Technical discussions for this Interest Group are conducted in <a href="https://www.w3.org/2015/Process-20150901/#confidentiality-levels">public</a>. Meeting minutes from teleconference and face-to-face meetings will be archived for public review, and technical discussions and issue tracking will be conducted in a manner that can be both read and written to by the general publi ...
+        <p id="public">Technical discussions for this Interest Group are conducted in <a href="https://www.w3.org/2015/Process-20150901/#confidentiality-levels">public</a>. Meeting minutes from teleconference and face-to-face meetings will be archived for public review, and technical discussions and issue tracking will be conducted in a manner that can be both read and written to by the general p ...
```
https://www.w3.org/2019/Process-20190301/

mlagally commented on charters/wot-ig-2019.html:
```diff
@@ -457,37 +393,29 @@ <h2 id="participation">
+        <p id="public">Technical discussions for this Interest Group are conducted in <a href="https://www.w3.org/2015/Process-20150901/#confidentiality-levels">public</a>. Meeting minutes from teleconference and face-to-face meetings will be archived for public review, and technical discussions and issue tracking will be conducted in a manner that can be both read and written to by the general p ...
 
-    <p>Information about the group (including details about deliverables, issues, actions, status, participants, and meetings) will be available from the <a href="http://www.w3.org/WoT/WG/">Web of Things Interest Group home page.</a></p>
+        <p>Information about the group (including details about deliverables, issues, actions, status, participants, and meetings) will be available from the <a href="http://www.w3.org/WoT/WG/">Web of Things Interest Group home page.</a></p>
```
https://www.w3.org/WoT/IG

mmccool commented on charters/wot-ig-2019.html:
```diff
@@ -167,278 +179,202 @@ <h1 id="title">Proposed Web of Things Interest Group Charter</h1>
+        </p>
+        <ul>
+          <li>use Cases and Requirements of different application scenarios</li>
+          <li>lifecycle management and Thing Templates</li>
```
maybe "clarify relationship with Digital Twins"

mmccool commented on charters/wot-ig-2019.html:
```diff
@@ -167,278 +179,202 @@ <h1 id="title">Proposed Web of Things Interest Group Charter</h1>
+        <p>
+          Further ideas for topics are expected from discussions and expert presentations of IoT projects,
+          including for example,
+          the <a href="https://www.w3.org/Data/events/data-ws-2019/cfp.html">Web Standardization for Graph Data</a>,
```
Dave Raggett: ref for data activity?

mkovatsc commented on charters/wot-ig-2019.html:
```diff
@@ -167,278 +179,202 @@ <h1 id="title">Proposed Web of Things Interest Group Charter</h1>
+        <ul>
+          <li>use Cases and Requirements of different application scenarios</li>
+          <li>lifecycle management and Thing Templates</li>
+          <li>profiles for zero-conf device onboarding to cloud services</li>
```
"Interoperability profiles for application domains" brings in the notion of siloing application domains again. I will drop this, as we should first get the mentioned "additional practical experience [...] in the field" to do something like this. Again, this does not put the topic automatically on the out-of-scope list...

mkovatsc commented on charters/wot-ig-2019.html:
```diff
@@ -167,278 +179,202 @@ <h1 id="title">Proposed Web of Things Interest Group Charter</h1>
-      	<p class="todo">In addition to the above catch-all reference to horizontal review which includes accessibility review, please check with chairs and staff contacts of the <a href="https://www.w3.org/WAI/APA/">Accessible Platform Architectures Working Group</a> to determine if an additional liaison statement with more specific information about concrete review issues is needed in the list be ...
-
-        <div>
+        <p>For all specifications, this Interest Group will seek <a href="https://www.w3.org/Guide/Charter.html#horizontal-review">horizontal review</a> for accessibility, internationalization, performance, privacy, and security with the relevant Working and Interest Groups, and with the <a href="https://www.w3.org/2001/tag/" title="Technical Architecture Group">TAG</a>. Invitation for review mus ...
```
Picked "deliverables"

mkovatsc commented:
I addressed the review comments.

mkovatsc commented:
Resolved merge conflict.

[New (updating, branch-based) preview](https://cdn.staticaly.com/gh/w3c/wot/mkovatsc-patch-ig-charter-2019/charters/wot-ig-2019.html?env=dev)

mkovatsc commented:
> Otherwise it sounds that there will be another WoT Scripting API

This is exactly what some people want. Our conclusion in Lyon was to continue the WG work, which will conclude as Note, for a low-level API based on Promises and to look into simpler, higher-level APIs that might be able to abstract the distributed nature behind simple JavaScript object properties and functions. Hence this wording.

mkovatsc commented:
Since the reviews are positive, I will merge to avoid people accidentally commenting on the version in the master.
